src/Microsoft.Health.Fhir.Core/Features/Operations/Import/ImportTaskMetricsNotification.cs

```diff
@@ -0,0 +1,49 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using Microsoft.Health.Fhir.Core.Features.Metrics;
+
+namespace Microsoft.Health.Fhir.Core.Features.Operations.Import
+{
+    public class ImportTaskMetricsNotification : IMetricsNotification
+    {
+        public ImportTaskMetricsNotification(
+            string id,
+            string status,
+            DateTimeOffset createdTime,
+            DateTimeOffset endTime,
+            long? dataSize,
+            long? succeedCount,
+            long? failedCount)
+        {
+            Id = id;
+            Status = status;
+            CreatedTime = createdTime;
+            EndTime = endTime;
+            DataSize = dataSize;
+            SucceedCount = succeedCount;
+            FailedCount = failedCount;
+        }
+
+        public string FhirOperation { get; }
+
+        public string ResourceType { get; }
```

Comment:
>ResourceType [](http://example.com/codeflow?start=22&length=12)

Since it's coming from interface, i would assume this is something we would like to populate.

Since import works for all resources, I would suggest to put `Resource` here.

Can be wrong, and it's totally ok to ignore this field.

Reply:
Import operation may contain many different types of resource, so ignore it maybe a good choice. Anyway, set it to null in the constructor.